manager_basic.go

```diff
@@ -58,9 +56,7 @@ func (m *basicProcessManager) CreateProcess(ctx context.Context, opts *CreateOpt
 	}
 
 	// TODO this will race because it runs later
```

Comment:
is this still true?

Reply:
This is still true since CreateProcess starts the process and if it completes before it reaches this line, the default trigger won't be registered. However, there isn't really anything to do about this so I can just remove it.